hardware/asic/README.md

```diff
@@ -62,9 +61,16 @@ sudo usermod -aG docker $USER
 ```bash
 newgrp docker
```

Comment:
I don't get this: the group has been created already. Isn't it created when you install docker?

Reply:
No sir the make in openlane will not do this .......... I was stuck at this point ......... it is important